src/AzureIoTHub.Portal/Server/Mappers/LoRaDeviceMapper.cs

```diff
@@ -118,6 +119,8 @@ public void UpdateTwin(Twin twin, LoRaDeviceDetails item)
 
             Helpers.DeviceHelper.SetDesiredProperty(twin, nameof(item.GatewayID), item.GatewayID);
 
+            Helpers.DeviceHelper.SetDesiredProperty(twin, nameof(item.ClassType), item.ClassType);
+
```

Comment:
This property is not supposed to be modified by the end user on device detail pages...
Consider removing this line